### PR TITLE
Preloader

### PR DESCRIPTION
**Pull request by milkcee12**
Preloader implemented and some Bootstrap fixes that happened after adding the React Router logic. 

https://user-images.githubusercontent.com/32281640/197591862-c20bd44c-8d5e-4339-a8d3-ef151f9b95fb.mp4

### CODE REVIEW

File: frontend/src/components/Preloader.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+            {/* Yoinked from Codepen: https://codepen.io/Treno/pen/vYJwQXw */}
+            <div className='wrapper'>
+                <span></span>
+                <span></span>
+                <span></span>
+                <span></span>
+                <span></span>
+                <span></span>
+                <span></span>
+                <span></span>
```
Comment (scottmai) [inline frontend/src/components/Preloader.js:15]: lol

File: frontend/src/index.scss [hunk 2]
```diff
@@ -537,3 +539,72 @@ body, html {
     height: 35px;
 }
 
+.preloader {
```
Comment (scottmai) [inline frontend/src/index.scss:542]: very fancy 👌

**Review by scottmai — APPROVED** [review]
noice 👌

File: frontend/src/components/Preloader.js (hunk 1 above)
Comment (milkcee12) [reply frontend/src/components/Preloader.js:15]: 🤣

File: frontend/src/index.scss (hunk 2 above)
Comment (milkcee12) [reply frontend/src/index.scss:542]: Codepen B)

**Pull request merged**